### DISCUSSION TITLE
Notify me when a new comment has been submitted

### DISCUSSION THREAD

**Discussion by Flor-e** [alkem-io/alkemio#847] [Ideas] [answered]
So I have made some comments in the challenge-centric hub. Some on the central hub, others on a specific challenge. I just realized that I do not receive a notification if anyone replies to it. That means I have to check back every now and then.

I would really like some way of knowing that someone replied to my comment, without receiving a notification for every single comment made. Because that could be a bit too much when things get busier on the platform. Some ideas:
- An actual reply function so you would only see direct replies on your message.
- A daily excerpt that tells me there are new comments in challenge a, b and c.

**Comment by techsmyth** [comment, marked as answer]
Hi Flori,

Re your questions: 
- comment notification in general:  this is a clear gap that we need to fill, it is captured - but will be the following sprint (starting Nov 7th) before we get to pick it up (unless something changes).  https://github.com/alkem-io/notifications/issues/150
- more targetted notifications: this is also one that is coming rapidly up in priority, it is clear that we need to add this. Question is how to prioritize this vs the other wish list items...but good to capture what kind of preferences you would want to have. This is also now captured (as an epic): https://github.com/alkem-io/alkemio/issues/849

Cheers, Neil

**Reply by Flor-e** [reply]
Maybe we can work out the "flexible notifications" as part of the "stimulating active contributions" (https://alkem.io/challengecentric/challenges/active-contributions/dashboard) challenge? I could add it as opportunity and work out a first draft. Will that help?
